dev/release-aux/release-version-fn.sh

```diff
@@ -102,6 +105,7 @@ set_version () {
             PRE_RELEASE_TAG="$PRE_LABEL$PRE_NUM"
             ;;
     esac
+    if [ -n "$PRE_RELEASE_TAG" ]; then _PRE_RELEASE_TAG="+${PRE_RELEASE_TAG}"; fi
```

Comment:
Here the prefix for `PRE_RELEASE_TAG` is a `"+"`. Earlier (line 58) a `"-"` is used.